<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Settings</class>
 <widget class="QDialog" name="Settings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>463</width>
    <height>440</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Downloader executable:</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLineEdit" name="edtDownloaderExe"/>
     </item>
     <item>
      <widget class="QPushButton" name="btnDownloaderExe">
       <property name="text">
        <string>Browse...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="lblOutputDir">
     <property name="text">
      <string>Output &amp;Directory:</string>
     </property>
     <property name="buddy">
      <cstring>edtOutputDir</cstring>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLineEdit" name="edtOutputDir"/>
     </item>
     <item>
      <widget class="QPushButton" name="btnBrowseOutput">
       <property name="text">
        <string>Browse...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="lblDownQueue">
     <property name="text">
      <string>Download &amp;Queue filename and location:</string>
     </property>
     <property name="buddy">
      <cstring>edtQueueFileDir</cstring>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLineEdit" name="edtQueueFileDir"/>
     </item>
     <item>
      <widget class="QPushButton" name="btnBrowseQueue">
       <property name="text">
        <string>Browse...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="lblDownFormats">
     <property name="text">
      <string>Default &amp;Formats:</string>
     </property>
     <property name="buddy">
      <cstring>txtDefFormats</cstring>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTextEdit" name="txtDefFormats"/>
   </item>
   <item>
    <widget class="QCheckBox" name="chkboxPreprocessLinks">
     <property name="text">
      <string>&amp;Preprocess youtube links (leave only v=.... arguments)</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>edtDownloaderExe</tabstop>
  <tabstop>btnDownloaderExe</tabstop>
  <tabstop>edtOutputDir</tabstop>
  <tabstop>btnBrowseOutput</tabstop>
  <tabstop>edtQueueFileDir</tabstop>
  <tabstop>btnBrowseQueue</tabstop>
  <tabstop>txtDefFormats</tabstop>
  <tabstop>chkboxPreprocessLinks</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Settings</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>266</x>
     <y>430</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Settings</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>334</x>
     <y>430</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
